frontend/src/metabase/visualizations/visualizations/Gauge/utils.ts

```diff
@@ -0,0 +1,13 @@
+export const getValue = (rows: unknown[][]) => {
+  const rawValue = rows[0] && rows[0][0];
+
+  if (rawValue === "Infinity") {
```

Comment:
Should this check be case insensitive, and are there other names to check for infinity? Or do we know for sure that it'll always a a serialized javascript "Infinity" value?

Reply:
Postgres and H2 support an `Infinity` value, mySQL (following the generic SQL spec) doesn't allow it at all. As far as I can tell it will always be capitalized from the DB itself.